src/polyglot-notebooks-vscode/README.md

```diff
@@ -10,18 +12,18 @@ Polyglot Notebooks are fully interoperable with Jupyter and support the `.ipynb`
 
 The following languages are supported by Polyglot Notebooks:
 
-| Language                                    | Variable sharing supported |
+| Language                                    | Variable sharing supported                             |
 |---------------------------------------------|-------------------------------------------------------|
-| C#                                          |        ✅                 |
-| F#                                          |        ✅                 |   
-| PowerShell                                  |        ✅                 |          
-| JavaScript                                  |        ✅                 |          
-| SQL                                         |        ✅                 |   
-| KQL ([Kusto Query Language](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/))    |        ✅       |       
-| HTML                                        |        ⛔                 |     
-| [Mermaid](https://mermaid.js.org/intro/)    |        ⛔                 |        
-| [Python](https://github.com/dotnet/interactive/blob/main/docs/jupyter-in-polyglot-notebooks.md)     |        ✅       |
-| [R](https://github.com/dotnet/interactive/blob/main/docs/jupyter-in-polyglot-notebooks.md)          |        ✅       |          
+| C#                                          |        ✅                                             |
+| F#                                          |        ✅                                             |
+| PowerShell                                  |        ✅                                             |
+| JavaScript                                  |        ✅                                             |
+| SQL                                         |        ✅                                             |
+| KQL ([Kusto Query Language](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/))    |        ✅       |
```

Comment:
nit: This may just be my OCD but the 2nd `|` char on this line appears to be misaligned. Only mentioning since you fixed up the indentation for the corresponding `|` chars on other lines 😄